### DISCUSSION TITLE
delta

### DISCUSSION THREAD

**Discussion by melmasri** [felixleopoldo/benchpress#21] [General]
https://github.com/felixleopoldo/benchpress/blob/26363f5b7832b6877de159b8de0571f42477a7a9/workflow/rules/algorithm_shell_commands.smk#L206

Why you have not assigned a delta parameter here. In this case, trilearn will set the delta parameter to 1, right?

**Comment by felixleopoldo** [comment]
The delta parameter is 1 by [default](https://github.com/felixleopoldo/trilearn/blob/96983e21eb1fe04c841384c1623a4afd6db66fb5/trilearn/pgibbs.py#L140), but it should also be possible to set it here.

**Reply by melmasri** [reply]
shoudn't we just assign it 3, since most models to that. I guess, we can leave it at that.
